### PR TITLE
Feature user interface milestone 2

### PR DESCRIPTION
**Pull request by davidauza-engineer**
Please let us know your comments about our work for milestone 2.

### CODE REVIEW

File: bin/main.rb [hunk 1]
```diff
@@ -1,2 +1,71 @@
...
+  i = 0
+  while i < matrix.length
+    matrix[i] = '-'
+    i += 1
+  end
+  matrix
+end
+
+# this method checks if the selected cell is empty: Empty means containing a '-'
+def check_matrix_cell(cell_to_check)
```
Comment (mcrd25) [inline bin/main.rb:14]: [OPTIONAL] A method like this would be a good candidate for Board class in next milestone

**Review by mcrd25 — CHANGES_REQUESTED** [review]
# Minor Changes Required 🙂 

Hi, @davidauza-engineer and @falguniislam 👋,

I really mean it when I say you did a good job for this milestone! 😄 

**There is only really one thing you need to address. Which are you do not consider whether the game is drawn or won after a player move and informing a person that there is a win or a draw.**
Other than that, you've done a good job.

### Required Changes
- Must consider in this milestone situation where a move results in a draw. This does not have to be code but at least considered in a comment as testing for draw and win would be more clearly defined in the next milestone.

***Please makes changes and submit another review request.
If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email.***

*Happy coding 😄,*
**[--Maya](https://github.com/mcrd25)**
[GitHub](https://github.com/mcrd25) [Twitter](https://twitter.com/mcrd25) [linkedin](https://www.linkedin.com/in/mayadouglas/) [GitLab](https://gitlab.com/mcrd25)

---
*P.S. all requested changes not marked by [OPTIONAL] or are not positive comments are **REQUIRED***

File: bin/main.rb (hunk 1 above)
Comment (davidauza-engineer) [reply bin/main.rb:14]: Thank you, we'll keep that in mind :)

**Comment by davidauza-engineer** [comment]
We added the requested changes, please let us know your feedback 👍

File: bin/main.rb [hunk 2]
```diff
@@ -1,2 +1,102 @@
...
+      puts "Please enter a number between 1 and 9:"
+    end
+  end
+  if i >= 4
+    if check_matrix_for_winner(player_id)
+      puts "\n\nGame Over!\n\nPlayer #{player_id} wins!"
+      break
+    else
+      if i == 8
+        puts "\n\nIt's a Draw!"
```
Comment (samgaco) [inline bin/main.rb:97]: :medal_sports:

**Review by samgaco — APPROVED** [review]
Hi @davidauza-engineer and @falguniislam :high_brightness: 

If this is the mockup I have high expectations for the real implementation already :laughing: 

The requirements in the previous review have been correctly implemented, therefore you can move on :+1: 

-------------------------------------
* ###### STATUS OF THE PROJECT: APPROVED :heavy_check_mark:

File: bin/main.rb (hunk 2 above)
Comment (davidauza-engineer) [reply bin/main.rb:97]: :D

**Comment by davidauza-engineer** [comment]
Thanks a lot for your review @mcrd25 @samgaco

File: bin/main.rb (hunk 2 above)
Comment (samgaco) [reply bin/main.rb:97]: :D

**Pull request merged**